clients/app/src/views/components/compositions/FanClubInfo/actions/CancelRequestActionSheet.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
```

Comment:
Can we combine this [CancelRequestActionSheet.tsx] and [RespondInvitationActionSheet.tsx] ?

Reply:
yeah, updated.